templates/Compensate_Image_Data.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>do we need the <code>sys.path.append('../')</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Since we're not starting these notebooks in Docker we'll need to explicitly add the path to <code>toffy</code> for these notebooks to be run directly.</p>